src/main/g8/scalafix/build.sbt

```diff
@@ -17,44 +19,93 @@ inThisBuild(
       )
     ),
     semanticdbEnabled := true,
-    semanticdbVersion := scalafixSemanticdb.revision,
-    scalacOptions ++= List(
-      "-P:semanticdb:synthetics:on"
-    )
+    semanticdbVersion := scalafixSemanticdb.revision
   )
 )
 
-publish / skip := true
+lazy val `$repo;format="normalize"$` = (project in file("."))
+  .aggregate(
+    rules.projectRefs ++
+      input.projectRefs ++
+      output.projectRefs ++
+      tests.projectRefs: _*
+  )
+  .settings(
+    publish / skip := true
+  )
 
-lazy val rules = project.settings(
-  moduleName := "scalafix",
-  libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % V.scalafixVersion
-)
+lazy val rules = projectMatrix
+  .settings(
+    moduleName := "scalafix",
+    libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % V.scalafixVersion
+  )
+  .defaultAxes(VirtualAxis.jvm)
+  .jvmPlatform(rulesCrossVersions)
 
-lazy val input = project.settings(
-  publish / skip := true
-)
+lazy val input = projectMatrix
+  .settings(
+    publish / skip := true
+  )
+  .jvmPlatform(
+    scalaVersions = rulesCrossVersions,
+    settings = Seq(
+      scalacOptions += "-P:semanticdb:synthetics:on"
+    )
+  )
+  .defaultAxes(VirtualAxis.jvm)
+  .jvmPlatform(scalaVersions = rulesCrossVersions :+ scala3Version)
```

Comment:
There are two calls to `.jvmPlatform` for `input`, with different `scalaVersions`. That seems off. Shouldn't the first one already mention `rulesCrossVersions :+ scala3Version` and the second one be removed?

Reply:
https://github.com/scalacenter/scalafix.g8/pull/19/commits/bfa2e49163dd2c715223815f832b60c5e08b8a95